.github/workflows/googletest.yaml

```
@@ -17,7 +17,17 @@ jobs:
       - name: Install dependencies
         run: sudo apt-get update && sudo apt-get install -y cmake g++ make
 
-      - name: Test
+      - name: ex00_Test
         run: |
             cd ex00
+            make test
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)